onnx/defs/op_annotation.h

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) Facebook Inc. and Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include <memory>
+#include <unordered_map>
+#include <unordered_set>
+
+#include "onnx/common/assertions.h"
+
+namespace ONNX_NAMESPACE {
+
+struct EnumHash {
+  template <typename T>
+  std::size_t operator()(T t) const {
+    return static_cast<std::size_t>(t);
+  }
+};
+
+enum OpAnnotationFlag : uint16_t {
```

Comment:
How about enum class?

Reply:
Also, could you add comments for each item to explain the meaning?

Reply:
Done.